backend/de.metas.cucumber/src/test/resources/de/metas/cucumber/features/accountingShippingNotification.feature

```diff
@@ -0,0 +1,486 @@
+      | pl_1       | ps_1                          | DE                        | EUR                 | price_list_name_04022023_1 | null            | false | false         | 2              | true         |
+      | pl_2       | ps_1                          | DE                        | EUR                 | price_list_name_04022023_2 | null            | true  | false         | 2              | true         |
+    And metasfresh contains M_PriceList_Versions
+      | Identifier | M_PriceList_ID.Identifier | Name                         | ValidFrom  |
+      | plv_1      | pl_1                      | purchaseOrder-PLV_04022023_1 | 2021-04-01 |
+      | plv_2      | pl_2                      | salesOrder-PLV_04022023_1    | 2021-04-01 |
+    And metasfresh contains M_ProductPrices
+      | Identifier | M_PriceList_Version_ID.Identifier | M_Product_ID.Identifier | PriceStd | C_UOM_ID.X12DE355 | C_TaxCategory_ID.InternalName |
+      | pp_1       | plv_1                             | p_1                     | 10.0     | PCE               | Normal                        |
+      | pp_2       | plv_2                             | p_1                     | 10.0     | PCE               | Normal                        |
+    And metasfresh contains C_BPartners:
+      | Identifier | Name                | OPT.IsVendor | OPT.IsCustomer | M_PricingSystem_ID.Identifier |
+      | bpartner_1 | BPartner_04022023_1 | Y            | Y              | ps_1                          |
+    And metasfresh contains C_BPartner_Locations:
+      | Identifier        | GLN           | C_BPartner_ID.Identifier | OPT.IsShipToDefault | OPT.IsBillToDefault |
+      | bpLocationDefault | 0123456789011 | bpartner_1               | Y                   | Y                   |
+    And metasfresh contains C_Orders:
+      | Identifier | IsSOTrx | C_BPartner_ID.Identifier | DateOrdered | OPT.C_PaymentTerm_ID | OPT.POReference | OPT.DocBaseType | OPT.M_PricingSystem_ID.Identifier |
+      | o_1        | N       | bpartner_1               | 2021-04-16  | 1000012              | po_ref_mock     | POO             | ps_1                              |
+    And metasfresh contains C_OrderLines:
+      | Identifier | C_Order_ID.Identifier | M_Product_ID.Identifier | QtyEntered |
+      | ol_1       | o_1                   | p_1                     | 20         |
+    When the order identified by o_1 is completed
+    And after not more than 30s, M_ReceiptSchedule are found:
+      | M_ReceiptSchedule_ID.Identifier | C_Order_ID.Identifier | C_OrderLine_ID.Identifier | C_BPartner_ID.Identifier | C_BPartner_Location_ID.Identifier | M_Product_ID.Identifier | QtyOrdered | M_Warehouse_ID.Identifier |
+      | receiptSchedule_04022023_1      | o_1                   | ol_1                      | bpartner_1               | bpLocationDefault                 | p_1                     | 20         | warehouseStd              |
```

Comment:
Why do we need this purchase part here?

Reply:
@cristinamghita 

<img width="632" alt="image" src="https://github.com/metasfresh/metasfresh/assets/103928467/9e9d736b-9ccd-42d9-898e-15c78a4828c4">